### PR TITLE
Add column model type into table models

### PR DESCRIPTION
**Pull request by lthoang**
### Related Issues

https://github.com/PreferredAI/cornac/issues/585

### Checklist:

- [ ] I have added tests.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated `README.md` (if you are adding a new model).
- [ ] I have updated `examples/README.md` (if you are adding a new example).
- [ ] I have updated `datasets/README.md` (if you are adding a new dataset).

### CODE REVIEW

**Review by hieuddo — APPROVED** [review]

**Comment by tqtg** [comment]
Let's keep this open for awhile

**Comment by tqtg** [comment]
I've tried to provide keywords that best describe the model type/family. If anyone has a better idea, please feel free to contribute.

**Comment by darrylong** [comment]
Was thinking as this is a long table with many columns, it could be a bit overwhelming.

Ideally we could have a sort feature (by year vs by model type) on this table. But upon checking, I think that is a limitation of MD files. 

Just throwing a few possibilities:
1. Create another "models.md" readme file to sort by model type. (more maintenance required)
2. Split this table into smaller tables based on categories (longer readme.md file)
3. Keep it as it is. I think the current table serves its purpose if generally users only care about using the most recent models.

Thanks everyone! :)

**Comment by tqtg** [comment]
> Was thinking as this is a long table with many columns, it could be a bit overwhelming.
> 
> Ideally we could have a sort feature (by year vs by model type) on this table. But upon checking, I think that is a limitation of MD files.
> 
> Just throwing a few possibilities:
> 
> 1. Create another "models.md" readme file to sort by model type. (more maintenance required)
> 2. Split this table into smaller tables based on categories (longer readme.md file)
> 3. Keep it as it is. I think the current table serves its purpose if generally users only care about using the most recent models.
> 
> Thanks everyone! :)

Agree that the current table is a bit long, though splitting into multiple tables will make things look very fragmented. Also, certain categories contain more models than others. I think it's best to keep one table for now until we find a better solution. I'm also thinking about experimenting with html table in markdown but not sure how much GitHub can render it.

File: README.md [hunk 1]
```diff
@@ -144,67 +144,68 @@ One important aspect of deploying recommender model is efficient retrieval via A
...
+| 2016 | [Collaborative Deep Ranking (CDR)](cornac/models/cdr), [paper](http://inpluslab.com/chenliang/homepagefiles/paper/hao-pakdd2016.pdf) | Content-Based / Text | [reqs](cornac/models/cdr/requirements.txt) | [cdr_exp.py](examples/cdr_example.py)
+|      | [Collaborative Ordinal Embedding (COE)](cornac/models/coe), [paper](http://www.hadylauw.com/publications/sdm16.pdf) | Collaborative Filtering | [reqs](cornac/models/coe/requirements.txt) |
+|      | [Convolutional Matrix Factorization (ConvMF)](cornac/models/conv_mf), [paper](http://uclab.khu.ac.kr/resources/publication/C_351.pdf) | Content-Based / Text | [reqs](cornac/models/conv_mf/requirements.txt) | [convmf_exp.py](examples/conv_mf_example.py)
+|      | [Learn to Rank user Preferences based on Phrase-level sentiment analysis across Multiple categories (LRPPM)](cornac/models/lrppm), [paper](https://www.yongfeng.me/attach/sigir16-chen.pdf) | Explainable | N/A | [lrppm_example.py](examples/lrppm_example.py)
```
Comment (tqtg) [inline README.md:181]: @lthoang any idea to shorten the title for this model?

Comment (lthoang) [reply README.md:181]: @tqtg How about "Learning to Rank Features for Recommendation
over Multiple Categories" as the paper title?

**Review by tqtg — APPROVED** [review]

**Pull request merged**